expr/functions/sortByName/function.go

```diff
@@ -35,19 +35,30 @@ func (f *sortByName) Do(ctx context.Context, e parser.Expr, from, until int64, v
 		return nil, err
 	}
 
-	natSort, err := e.GetBoolNamedOrPosArgDefault("natural", 1, false)
+	isNatSort, err := e.GetBoolNamedOrPosArgDefault("natural", 1, false)
+	if err != nil {
+		return nil, err
+	}
+
+	isReverseSort, err := e.GetBoolNamedOrPosArgDefault("reverse", 2, false)
```

Comment:
It makes sense to update a description

Reply:
Parameter was already in description, but wasn't actually used.